test/data/dell_2_numa/machineinfo.json

```diff
@@ -1 +1,1565 @@
+{
+    "timestamp": "2000-01-01T00:00:00Z",
+    "vendor_id": "AuthenticAMD",
```

Comment:
from where does this come from?

Reply:
its arch dependent, it comes from the ci cluster.

Reply:
I'm pretty sure the cluster is Intel, though